### PR TITLE
address anonymous functions

### LINKED ISSUE #30: Unnecessary closure/IIFE?
In the advice about [private members](https://github.com/ryanmcdermott/clean-code-javascript/blob/master/README.md#make-objects-have-private-members), what's the point of the extra IIFE over `Employee` definition?

```js
var Employee = (function() {
  function Employee(name) {
    this.getName = function() {
      return name;
    };
  }

  return Employee;
}());
```

Since there are no extra variables in the outer function, this snippet could as well read:

```js
function Employee(name) {
  this.getName = function() {
    return name;
  };
 }
```

### PR DESCRIPTION
**Pull request by vsemozhetbyt**
(Here comes a bunch of PRs after a code fragments mass test with ESLint checks and Node.js runnings.)

1. Add function name if it can't be inferred from the context. See [func-names](http://eslint.org/docs/rules/func-names).
2. Replace anonymous functions with arrow functions when appropriate. See [prefer-arrow-callback](http://eslint.org/docs/rules/prefer-arrow-callback).
3. Replace anonymous functions with method shorthand notation when appropriate. See [object-shorthand](http://eslint.org/docs/rules/object-shorthand).
4. Remove a redundant IIFE. Fixes #30 temporarily.

### CODE REVIEW

**Comment by ryanmcdermott** [comment]
Rebase with master!

**Comment by vsemozhetbyt** [comment]
@ryanmcdermott Done.

File: README.md [hunk 1]
```diff
@@ -1296,7 +1292,7 @@ class DOMTraverser {
 const $ = new DOMTraverser({
   rootNode: document.getElementsByTagName('body'),
   options: {
-    animationModule: function() {}
+    animationModul() {}
```
Comment (ryanmcdermott) [inline README.md]: Looks like this is misspelled.

Comment (vsemozhetbyt) [inline README.md]: Sorry, corrected.

**Comment by ryanmcdermott** [comment]
Great! Thanks for fixing that Private Members issue too, that was a silly mistake on my part.

**Pull request merged**